recipes/syngap/meta.yaml

```diff
@@ -83,7 +83,8 @@ test:
     - genepair
     - evi
     - eviplot
-  commands: 
+  commands:
+    - df -h
```

Comment:
```suggestion
    - df -h
    - du -sch $CONDA_PREFIX
```

Reply:
Thanks for review!
Did these modifications to build.sh and meta.yaml work?

Reply:
No. The changes show that 5Gb disk space is taken by the Conda environment (dependencies). 
65Gb are taken by something else (I have no idea what exactly)
3.4Gb are free.

The build does not fail due to out of disk space!
At 47-48 minute it ends with `This job was abandoned`. I have seen this behavior in other PRs too but so far I don't have an idea why.

Reply:
Oh no. That's so wired.
The previous PR of this package wasn't that much different to this one.